std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -525,6 +530,15 @@ struct KRRegion(ParentAllocator = NullAllocator)
         return null;
     }
 
+    private void ensureCorrectSize()
+    {
+        import std.math : abs;
+
+        if (root && (abs(payload.length - root.size) < Node.sizeof)) {
```

Comment:
For completion, the fixes to `Node.adjacent` and `Node.coalesce` can't hurt (not sure whether they help all cases but are clearly buggy at the moment).